packages/components/src/form-toggle/stories/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import FormToggle from '../';
+
+export default { title: 'Form Toggle', component: FormToggle };
```

Comment:
```suggestion
export default { title: 'FormToggle', component: FormToggle };
```

Reply:
I don't have perms to commit myself. We can tackle it separately. It's just me looking for consistency in naming :)

Reply:
Following up on this! We've tidied up Storybook a bit. For components, we would label it like:

`Components|FormToggle`

Which automatically categorizes it in Storybook's sidebar